inc/tainetworkif.h

```diff
@@ -396,6 +431,38 @@ typedef enum _tai_network_interface_attr_t
      */
     TAI_NETWORK_INTERFACE_ATTR_CH1_FREQ,
 
+    /**
+     * @brief Rx OTU Status with Open ROADM Multi-Source Agreement
+     *
+     * @type #tai_network_interface_rx_otu_status_t
```

Comment:
Is this status only show one alarm at a specific time?
If it can include several alarms at the same time, you need to make this as a list of enums like [this](https://github.com/Telecominfraproject/oopt-tai/blob/master/inc/tainetworkif.h#L160)

Reply:
Same for other attributes.